src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -186,39 +186,41 @@ private static bool TryGetTextForOperator(SyntaxToken token, Document document,
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.ColonColonToken))
+            if (token.IsKind(SyntaxKind.InterpolatedStringStartToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringEndToken) ||
+                token.IsKind(SyntaxKind.InterpolatedStringTextToken))
             {
-                text = "::_CSharpKeyword";
+                text = "$_CSharpKeyword";
                 return true;
             }
 
-            if (token.Kind() == SyntaxKind.ColonToken && token.Parent is NameColonSyntax)
+            if (token.IsVerbatimStringLiteral())
             {
-                text = "cs_namedParameter";
+                text = "@_CSharpKeyword";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.QuestionToken) && token.Parent is ConditionalExpressionSyntax)
+            if (token.IsKind(SyntaxKind.ColonColonToken))
             {
-                text = "?_CSharpKeyword";
+                text = "::_CSharpKeyword";
                 return true;
             }
 
-            if (token.IsKind(SyntaxKind.EqualsGreaterThanToken))
+            if (token.Kind() == SyntaxKind.ColonToken && token.Parent is NameColonSyntax)
             {
-                text = "=>_CSharpKeyword";
+                text = "cs_namedParameter";
```

Comment:
Probably could make this `namedParameter_CSharpKeyword` for consistency. It's already mapped in the docs.